docs/source/user_manual/results_filtering.rst

```diff
@@ -19,9 +22,15 @@ Relevant light curve filtering parameters include:
  * ``max_lh`` - The maximum likelihood to keep.
  * ``sigmaG_lims`` - The percentiles for sigmaG filtering (default of [25, 75]).
 
-The stamp filtering stage is only applied if the ``do_stamp_filter`` parameter is set to True. This stage by creating a single stamp representing the sum, mean, or median of pixel values for the stamps at each time step. The stamp type is defined by the ``stamp_type`` parameter and can take on values ``median``, ``cpp_median``, ``cpp_mean``, ``parallel_sum``, or ``sum``. All of the stamp types drop masked pixels from their computations. The mean and median sums are computed over only the valid time steps from the light curve filtering phase (dropping stamps with outlier fluxes). The sum coadd uses all the time steps regardless of the first phase of filtering.
+Stamp Filtering
+---------------
+
+The stamp filtering stage is only applied if the ``do_stamp_filter`` parameter is set to True. This stage by creating a single stamp representing the sum, mean, or median of pixel values for the stamps at each time step. The stamp type is defined by the ``stamp_type`` parameter and can take on values ``median``, ``mean``, or ``sum``. All of the stamp types drop masked pixels from their computations. The mean and median sums are computed over only the valid time steps from the light curve filtering phase (dropping stamps with outlier fluxes). The sum coadd uses all the time steps regardless of the first phase of filtering.
```

Comment:
I think that the beginning of the second sentence is suppose to be "This stage creates ...". Not 100% sure.
```suggestion
The stamp filtering stage is only applied if the ``do_stamp_filter`` parameter is set to True. This stage by creates a single stamp representing the sum, mean, or median of pixel values for the stamps at each time step. The stamp type is defined by the ``stamp_type`` parameter and can take on values ``median``, ``mean``, or ``sum``. All of the stamp types drop masked pixels from their computations. The mean and median sums are computed over only the valid time steps from the light curve filtering phase (dropping stamps with outlier fluxes). The sum coadd uses all the time steps regardless of the first phase of filtering.
```